app/complaint/complaint_schema.py

```diff
@@ -0,0 +1,11 @@
+
+complaint_count_request = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "complaint count request schema",
+    "type": "object",
+    "title": "Complaint count request",
+    "properties": {
+        "start_date": {"type": ["string", "null"], "format": "date"},
```

Comment:
Is date a default format type in jsonschema?

Reply:
Yes - there's a test here in case anything to do with the date schema validation changes

https://github.com/alphagov/notifications-api/blob/531a7a5aaa29c0f5eb422cb523b5b171ac2a56e7/tests/app/complaint/test_complaint_rest.py#L55-L64